File: tests/breve/perf/test.perf.01.loops.py
import time

print("Doing 100 million loops...")

start = time.time()

print("Loop iteration #0")

for i in range(int(10e8)):
	if not int((i+1) % 10e7):
		print(f"Loop iteration #{i+1}")

total_time = (time.time() - start)

print("\nResult::{")
print(f"  total-time: {total_time}")
print("}")

File: tests/breve/perf/test.perf.01.loops.lua
print("Doing 100 million loops...")

local start = os.clock()

print("Loop iteration #0")

for i=1,10e8 do
	if (i % 10e7) == 0 then
		print("Loop iteration #" .. i)
	end
end

local total = (os.clock() - start)

print("\nResult::{")
print("  total-time: " .. total)
print("}")